src/plugins/intel_cpu/src/config.cpp

```diff
@@ -108,71 +110,76 @@ void Config::readProperties(const std::map<std::string, std::string> &prop) {
                            << ". Expected only true/false." << std::endl;
             }
-        } else if (key == CPUConfigParams::KEY_CPU_SPARSE_WEIGHTS_DECOMPRESSION_RATE) {
+        } else if (key == ov::intel_cpu::sparse_weights_decompression_rate.name()) {
             float val_f = 0.0f;
             try {
                 val_f = std::stof(val);
             } catch (const std::exception&) {
-                IE_THROW() << "Wrong value for property key " << CPUConfigParams::KEY_CPU_SPARSE_WEIGHTS_DECOMPRESSION_RATE
-                                    << ". Expected only float numbers";
+                IE_THROW() << "Wrong value for property key " << ov::intel_cpu::sparse_weights_decompression_rate.name()
+                           << ". Expected only float numbers";
             }
             if (val_f < 0.f || val_f > 1.f) {
-                IE_THROW() << "Wrong value for property key " << CPUConfigParams::KEY_CPU_SPARSE_WEIGHTS_DECOMPRESSION_RATE
-                                    << ". Sparse rate must be in range [0.0f,1.0f]";
+                IE_THROW() << "Wrong value for property key " << ov::intel_cpu::sparse_weights_decompression_rate.name()
+                           << ". Sparse rate must be in range [0.0f,1.0f]";
             } else {
                 fcSparseWeiDecompressionRate = val_f;
             }
-        } else if (key == PluginConfigParams::KEY_PERF_COUNT) {
-            if (val == PluginConfigParams::YES) collectPerfCounters = true;
-            else if (val == PluginConfigParams::NO) collectPerfCounters = false;
+        } else if (key == ov::enable_profiling.name()) {
+            if (val == InferenceEngine::PluginConfigParams::YES)
+                collectPerfCounters = true;
+            else if (val == InferenceEngine::PluginConfigParams::NO)
+                collectPerfCounters = false;
             else
-                IE_THROW() << "Wrong value for property key " << PluginConfigParams::KEY_PERF_COUNT
-                                   << ". Expected only YES/NO";
-        } else if (key == PluginConfigParams::KEY_EXCLUSIVE_ASYNC_REQUESTS) {
-            if (val == PluginConfigParams::YES) exclusiveAsyncRequests = true;
-            else if (val == PluginConfigParams::NO) exclusiveAsyncRequests = false;
+                IE_THROW() << "Wrong value for property key " << ov::enable_profiling.name()
+                           << ". Expected only YES/NO";
+        } else if (key == ov::exclusive_async_requests.name()) {
+            if (val == InferenceEngine::PluginConfigParams::YES)
+                exclusiveAsyncRequests = true;
+            else if (val == InferenceEngine::PluginConfigParams::NO)
+                exclusiveAsyncRequests = false;
             else
-                IE_THROW() << "Wrong value for property key " << PluginConfigParams::KEY_EXCLUSIVE_ASYNC_REQUESTS
+                IE_THROW() << "Wrong value for property key " << ov::exclusive_async_requests.name()
```

Comment:
can we refactor CPU config as a separate PR? I suppose it's quite self-sufficient functionality and can be extracted into a separate PR for easy review.

Reply:
yes, CPU config will be another following PR, current PR is only for base interface migration.